pyvrp/cpp/ProblemData.h

```diff
@@ -415,6 +419,12 @@ class ProblemData
      * max_reloads
      *     Maximum number of reloads the vehicle may perform on a route.
      *     Unconstrained if not explicitly provided.
+     * max_overtime
+     *     Maximum allowed overtime, on top of the :py:attr:`~shift_duration`.
+     *     Default 0, that is, overtime is not allowed.
+     * unit_overtime_cost
+     *     Cost of a unit of overtime. This is in addition to the regular
+     *     :py:attr:`~unit_duration_cost` of route durations. Default 0.
```

Comment:
Not something actionable, but I think that `unit_overtime_cost` can be slightly misleading. The relationship between overtime and duration is not immediately obvious to users. For example, if regular rates are €20 and overtime rates are €30, I can imagine that users would put 20 and 30 for `unit_duration_cost` and `unit_overtime_cost`, instead of 20 and 10. 

I think this can be easily addressed by an example. Should I make one after merging this PR?